docs/source/about/guides/linux.rst

```diff
@@ -0,0 +1,459 @@
+	I suppose it's not possible for SSH to trigger a udev rule.
+
+**Setup Script**
+
+This script will take care of any precondtions prior to starting up sunshine.
+Create a script named something like ``sunshine-setup.sh``:
+
+.. code-block:: bash
+
+	#!/bin/bash
+	chown <user>:<user> /dev/uinput
+
+	# Optional
+	# blocks wifi, so ethernet is used
+	# use rfkill list to get the id of the Wiresless LAN
+	# rfkill block <wireless_lan_index>
+
+**Sudo Configuration**
+
+We will manually change the permissions of ``/dev/uinput`` using ``chown``.
+You need to use ``sudo`` to make this change, so add/update the entry in ``/etc/sudoers.d/<user>``
+
+.. caution::
+	Do so at your own risk! It is more secure to give sudo and no password prompt to a single script, than a generic executable like chown.
+
+.. warning::
+	Be very careful of messing this config up. If you make a typo, *YOU LOSE THE ABILITY TO USE SUDO*. Fortunately, your system is not borked,
+	you will need to login as root to fix the config. You may want to setup a backup user / SSH into the host as root to fix the config if this happens. Otherwise you will need to plug your machine back into a monitor and login as root to fix this. To enable root login over SSH edit your SSHD config, and add ``PermitRootLogin yes``, and restart the SSH server.
+
+Replace ``<user>`` with your user and add the absolute path to your script:
+
+.. code-block::
+
+	<user> ALL=(ALL:ALL) ALL, NOPASSWD: /path/to/sunshine-setup.sh
+
+These changes allow the script to use sudo without being prompted with a password.
+
+E.g. ``sudo /path/to/sunshine-setup.sh``
+
+
+Stream Launcher Script
+++++++++++++++++++++++
+
+This is the main entrypoint script that will run the sunshine-setup script, start up X server, and sunshine. *This is your wrapper entrypoint script that the ssh client will run to start streaming with sunshine*.
+
+
+**Sunshine Startup Script**
+
+This guide will refer to this script as ``~/scripts/sunshine.sh``. The setup script will be referred as ``~/scripts/sunshine-setup.sh``
+
+.. code-block:: bash
+
+    #!/bin/bash
+
+    export DISPLAY=:0
+
+    # Check existing X server
+    ps -e | grep X >/dev/null
+    [[ ${?} -ne 0 ]] && {
+	  echo "Starting X server"
+	  startx &>/dev/null &
+	  [[ ${?} -eq 0 ]] && {
+	    echo "X server started successfully"
+	  } || echo "X server failed to start"
+    } || echo "X server already running"
+
+    # Check if sunshine is already running
+    ps -e | grep -e .*sunshine$ >/dev/null
+    [[ ${?} -ne 0 ]] && {
+	  sudo ~/scripts/sunshine-setup.sh
+	  echo "Starting Sunshine!"
+	  sunshine > /dev/null &
+	  [[ ${?} -eq 0 ]] && {
+	    echo "Sunshine started successfully"
+	  } || echo "Sunshine failed to start"
+    } || echo "Sunshine is already running"
+
+    # Add any other Programs that you want to startup automatically
+    # E.g.
+    # steam &> /dev/null &
+    # firefox &> /dev/null &
+    # kdeconnect-app &> /dev/null &
+
+----
```

Comment:
```suggestion
```

If this is useful, leave it... just never saw it before in rst.

Reply:
This syntax represents [horizontal rules](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr)
>The \<hr\> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a thematic break between paragraph-level elements: for example, a change of scene in a story, or a shift of topic within a section.

I got rid of the other one since I wasn't being consistent with change of topics of the subsections. But I think it makes sense here from shifting from Sunshine host setup to SSH client setup.